config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -17,13 +17,20 @@ periodics:
         args:
         - -c
         - |
-          set -euo; \
-          export PATH=$PATH:$GOPATH/bin; \
-          mkdir -p "${ARTIFACTS}"; \
-          pushd "$ARTIFACTS"; \
-          go install github.com/kubernetes-sigs/depstat@latest; \
-          popd; \
-          depstat stats --json | tee "${ARTIFACTS}/stats-base.json";
+          set -euo pipefail
```

Comment:
test-infra needs a linter for "size of inline prow bash" 😂 

I think this might not be over the limit yet but .. it's growing 👀